data/build.gradle

```diff
@@ -10,23 +10,12 @@ android {
     defaultConfig {
         javaCompileOptions {
 
-            buildConfigField javaTypes.STRING, "TRANSACTION_SERVICE_URL", asString(getKey("TRANSACTION_SERVICE_URL", "https://safe-transaction.rinkeby.gnosis.io/api/"))
-            buildConfigField javaTypes.STRING, "RELAY_SERVICE_URL", asString(getKey("RELAY_SERVICE_URL", "https://safe-relay.rinkeby.gnosis.pm/api/"))
-
             annotationProcessorOptions {
                 arguments = ["room.schemaLocation": "$projectDir/schemas".toString()]
             }
         }
     }
 
-    buildTypes {
```

Comment:
just add other flavors here explicitly and place buildConfigFields with endpoint addresses there